tests/test_excluded_packages.py

```diff
@@ -0,0 +1,31 @@
+import os
+import subprocess
+
+from common.report_common import assert_story_points_from_report_file
+
+
+def test_excluded_packages(analysis_data):
+    application_data = analysis_data['jee_example_app']
+
+    windup_path = os.getenv('WINDUP_CLI_PATH')
+    report_path = os.getenv('REPORT_OUTPUT_PATH')
+    binary_path = os.path.join(
+        os.getenv('PROJECT_PATH'),
+        'fixtures/applications',
+        application_data['file_name']
+    )
+
+    command = windup_path + ' --output ' + report_path + ' --input ' + binary_path + ' --target ' + \
```

Comment:
Can we create a utility method to generate command ?
By default it will generate a basic command with cli , input , output and it can take any additional optional parameters like in this case "excludepackage"





Example 
    


    generate_command(parameter?, inputForParameter?){
    if parameter :
        command = command + "--" +parameter    
                      +  inputForparamater
     return command;
     }




Pardon the syntax , it is mixed with cypress :)

Reply:
Yes, you're right.

Please take a look at the last commit, I created a pretty generic method with the main arguments as standard parameters and a dictionary argument for additional command parameters.